osbs/utils/__init__.py

```diff
@@ -734,6 +734,10 @@ def __init__(self):
         self.regex = r' - '.join((r'^.+', USER_WARNING_LEVEL_NAME, r'(\{.*\})$'))
         self._user_warnings = set()
 
+    @property
+    def UW_list(self):
```

Comment:
can we call it just `warnings` ?

Reply:
I was thinking and it may be cool to have an iterator to iterate over messages as part of the class (`__iter__` method).